requirements_all.txt

```diff
@@ -307,7 +307,7 @@ https://github.com/soldag/pyflic/archive/0.4.zip#pyflic==0.4
 https://github.com/tfriedel/python-lightify/archive/1bb1db0e7bd5b14304d7bb267e2398cd5160df46.zip#lightify==1.0.5
 
 # homeassistant.components.tado
-https://github.com/wmalgadey/PyTado/archive/0.1.10.zip#PyTado==0.1.10
+https://github.com/wmalgadey/PyTado/archive/0.2.1.zip#PyTado==0.2.1
```

Comment:
On PyPI there is [1.0.3](https://pypi.python.org/pypi/pytado/1.0.3) available. Any reason to use a fork instead of the upstream project?